README.md

```diff
@@ -39,18 +37,22 @@ ran against this extention library. If you are porting your code to this codebas
 
 Some of the major differences between this library and Numenta's extension library are the following:
 
- * Support for Python 2.7 and Python 3.x (Only Python 3.x under windows)
+ * Support for Python 3 and Python 2.7 (Only Python 3 under windows)
  * Support for Linux, OSx, and Windows MS Visual Studio 2017
- * Support for C++11, C++17 
+ * Support for C++11 through C++17 
  * Replaced SWIG with PyBind11 for Python interface.
  * Removed CapnProto serialization.  It was prevasive and complicated the code considerably. It was replaced 
  with simple binary streaming serialization in C++ library.
  * Many code optimizations, modernization (Spatial Pooler shares optimized Connections backend with Temporal memory) 
  * Modular structure
  * Interfaces & API stabilization, making it easier for developers & researchers to use our codebase
  * Much easier installation (reduced dependencies, all are handeled by CMake) 
- * static and shared lib files for use with C++ applications.
-
+ * Static and shared lib files for use with C++ applications.
+ * New and Improved Algorithms:
+   - Sparse Distributed Representations
+   - Anomaly Likelihood
+   - Backtracking Temporal Memory
+   - Significantly faster Spatial Pooler and Connections
 
 ## Installation 
```

Comment:
make a link/mention from Readme.md to Release_Goalds.md, 
I would even remove the latter file and place its text as a section here in Readme.

Reply:
..ok, the link to Goals is there